packages/presentation/package.json

```diff
@@ -54,6 +54,7 @@
     "@hcengineering/client": "^0.6.18",
     "@hcengineering/collaborator-client": "^0.6.4",
     "fast-equals": "^5.2.2",
+    "marked": "^15.0.7",
```

Comment:
We need to use our markdown converter I think, so you only need to convert markdown used in github service or others and put it into Node as node.

Reply:
Yes, this option was a quick fix for demo, but now it's not actual.
I will create a new PR with markdown converter